pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -427,6 +427,17 @@ public enum PGProperty {
       "5432",
       "Port of the PostgreSQL server (may be specified directly in the JDBC URL)"),
 
+  /**
+   * Controls the styles of parameter placeholders accepted in prepared statements. The default value of {@code none}
+   * disables non-standard placeholders.
+   */
+  PLACEHOLDER_STYLES(
+      "placeholderStyles",
```

Comment:
I would say `placeholderStyle` sounds better.

For instance, I find `placeholderStyle=named` sound better than `placeholderStyles=named`. The naming of `placeholderStyles` sounds to me as if a single placeholder could have several styles.

WDYT?